pages/hub/insights/index.tsx

```diff
@@ -27,7 +62,9 @@ const InsightsHub: WithPageLayout = () => {
 
   function handleView() {
     const insight = data.slice(0, 1).shift();
-    router.push(`/pages/${user?.user_metadata.user_name}/${insight!.id}/dashboard`);
+    router.push(
+      `/pages/${user ? user.user_metadata.user_name : "anonymous"}/${insight ? insight!.id : "734"}/dashboard`
```

Comment:
What is 734?  The prod ID for the featured insight page? We shouldn't hardcode that here

Reply:
Agree with @brandonroberts about all the hard-coding. I wonder if we create a demo account that can't log in that has some insights instead?

Reply:
+1 - clicking this link takes me back to `/hub/insights` so it maybe isn't there?

Reply:
The Idea i had in mind was create a similar insight page with same repos and name in beta and store the id in `.env`. This way we can have it work for both dev and prod environment. open to suggestions on this also

Reply:
> What is 734? The prod ID for the featured insight page? We shouldn't hardcode that here

This was intended to be stored in `.env` file.